refact_webgui/webgui/static/tab-finetune.html

```diff
@@ -191,7 +191,7 @@ <h5 class="modal-title" id="urlModalLabel">Launch Finetune</h5>
               <label class="form-label">Do you have a large codebase?</label>
               <div class="btn-group" role="group" aria-label="basic radio toggle button group">
                 <input type="radio" class="btn-check" name="trainable_embeddings" id="trainable_embeddings1" value="1" autocomplete="off">
-                <label class="btn btn-outline-primary" for="trainable_embeddings1">Yes, train embeddings</label>
+                <label class="btn btn-outline-primary" for="trainable_embeddings1">Yes, train embeddings (unstable with vllm)</label>
```

Comment:
if vllm is the new default, then disable this completely, don't say it's unstable?

Reply:
going to be changed in the next version when vllm actually become a new default provider